pkg/webreg/zz_generated.ci_operator_reference.go

```diff
@@ -0,0 +1,1210 @@
+	"              - default: \"\"\n" +
+	"                documentation: ' '\n" +
+	"                name: ' '\n" +
+	"            from: ' '\n" +
+	"            from_image:\n" +
+	"                # LiteralTestStep is a full test step definition.\n" +
+	"                as: ' '\n" +
+	"                name: ' '\n" +
+	"                namespace: ' '\n" +
+	"                tag: ' '\n" +
+	"            optional_on_success: false\n" +
+	"            # Reference is the name of a step reference.\n" +
+	"            ref: \"\"\n" +
+	"            # Resources defines the resource requirements for the step.\n" +
+	"            resources:\n" +
+	"                # LiteralTestStep is a full test step definition.\n" +
+	"                limits:\n" +
+	"                    # LiteralTestStep is a full test step definition.\n" +
+	"                    \"\": \"\"\n" +
+	"                requests:\n" +
+	"                    # LiteralTestStep is a full test step definition.\n" +
+	"                    \"\": \"\"\n" +
+	"            termination_grace_period_seconds: 0\n" +
+	"        # Test is the array of test steps that define the actual test.\n" +
+	"        test:\n" +
+	"          # LiteralTestStep is a full test step definition.\n" +
+	"          - active_deadline_seconds: 0\n" +
+	"            artifact_dir: ' '\n" +
+	"            as: ' '\n" +
+	"            # Chain is the name of a step chain reference.\n" +
+	"            chain: \"\"\n" +
+	"            # Cli is the (optional) name of the release from which the `oc` binary\n" +
+	"            # will be injected into this step.\n" +
+	"            cli: ' '\n" +
+	"            commands: ' '\n" +
+	"            credentials:\n" +
+	"              # LiteralTestStep is a full test step definition.\n" +
+	"              - mount_path: ' '\n" +
+	"                name: ' '\n" +
+	"                namespace: ' '\n" +
+	"            dependencies:\n" +
+	"              # LiteralTestStep is a full test step definition.\n" +
+	"              - env: ' '\n" +
+	"                name: ' '\n" +
+	"            env:\n" +
+	"              # LiteralTestStep is a full test step definition.\n" +
+	"              - default: \"\"\n" +
+	"                documentation: ' '\n" +
+	"                name: ' '\n" +
+	"            from: ' '\n" +
+	"            from_image:\n" +
+	"                # LiteralTestStep is a full test step definition.\n" +
+	"                as: ' '\n" +
+	"                name: ' '\n" +
+	"                namespace: ' '\n" +
+	"                tag: ' '\n" +
+	"            optional_on_success: false\n" +
+	"            # Reference is the name of a step reference.\n" +
+	"            ref: \"\"\n" +
+	"            # Resources defines the resource requirements for the step.\n" +
+	"            resources:\n" +
+	"                # LiteralTestStep is a full test step definition.\n" +
+	"                limits:\n" +
+	"                    # LiteralTestStep is a full test step definition.\n" +
```

Comment:
This don't look quite right

Reply:
How so?

Reply:
This string "LiteralTestStep..." is repeated like 400 times on unrelated fields in here

Reply:
Hm, that doc comes from a promoted field: https://github.com/openshift/ci-tools/blob/581ca445940d60c43bce33451236e3ce8d134cab/pkg/api/types.go#L650

Seems like a bug in genyaml. Do you mind merging this already withotu advertising it and I will look into the genyaml stuff later?